app/controllers/drafts_controller.rb

```diff
@@ -1,6 +1,6 @@
 # :nodoc:
-class DraftsController < ManageMetadataController
-  include ManageMetadataHelper
-  
+class DraftsController < ManageCollectionsController
```

Comment:
@rdcody, @abbottry, it seems a little weird to me to have the DraftsController inherit from a `ManageCollectionsController`. This would mean that VariableDrafts and eventually ServiceDrafts would also be inheriting from that controller. I know we need to change the UI to represent 'Manage Collections' instead of 'Manage Metadata' and to create space for a 'Manage Variables'. But maybe we should still have a `ManageMetadataController` that sits above both of those controllers as well as the `DraftsController`?